.github/workflows/lint-swagger.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: Swagger Editor Validator Remote
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
This still runs the old linter, if you can, sync the structure with analytics:
- lint-swagger.yml only (no redocly...)